### PR TITLE
Improved the event subscriber that listens to database exceptions

### PR DESCRIPTION
**Pull request by javiereguiluz**
This finishes #431 by doing some renaming and minor tweaks.

### CODE REVIEW

File: src/AppBundle/EventListener/CheckRequirementsSubscriber.php [hunk 1]
```diff
@@ -54,9 +50,9 @@ public static function getSubscribedEvents()
         return [
             // Exceptions are one of the events defined by the Console. See the
             // rest here: http://symfony.com/doc/current/components/console/events.html
-            ConsoleEvents::EXCEPTION => 'handleDatabaseExceptions',
-            // See: http://api.symfony.com/3.2/Symfony/Component/HttpKernel/KernelEvents.html
-            KernelEvents::EXCEPTION => 'onKernelException',
+            ConsoleEvents::EXCEPTION => 'handleConsoleException',
+            // See: http://api.symfony.com/master/Symfony/Component/HttpKernel/KernelEvents.html
```
Comment (stof) [inline src/AppBundle/EventListener/CheckRequirementsSubscriber.php]: can we use ``current`` on the API doc or no ?

**Comment by stof** [comment]
👍

File: src/AppBundle/EventListener/CheckRequirementsSubscriber.php (hunk 1 above)
Comment (javiereguiluz) [inline src/AppBundle/EventListener/CheckRequirementsSubscriber.php]: It was my first try, but ...

api_current

Comment (stof) [inline src/AppBundle/EventListener/CheckRequirementsSubscriber.php]: then fine with using master

**Pull request merged**